src/MakersOfDenmark.Domain/Models/Tool.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace MakersOfDenmark.Domain.Models
+{
+    public class Tool
+    {
+        public Guid Id { get; set; }
```

Comment:
Meget lille ændring, men Id kunne måske også her være int i stedet for Guid

Reply:
+1